python-modules.sh

```diff
@@ -34,6 +34,8 @@ mkdir -p $PYTHON_MODULES_INSTALLROOT
 # See also:
 #   https://github.com/scikit-garden/scikit-garden/issues/23
 grep RootInteractive requirements.txt && env PYTHONUSERBASE="$PYTHON_MODULES_INSTALLROOT" pip3 install --user -IU numpy
+# Do not move cython from 0.29.06 for now since 3.0.0rc1 breaks on GPU
```

Comment:
For reference, it doesn't only fail in the GPU CI, but for me it also fails in the alisw/slc7-builder when I do not use precompiled builds.

Reply:
Ok. Does this fix the issue for you?